tests/nightly/test_large_array.py

```diff
@@ -1212,6 +1214,80 @@ def test_full():
     assert a[-1][-1] == 3
 
 
+def test_hyperbolic():
+    def test_arccosh(a):
```

Comment:
nit if CI fails: could you add a comment about testing only the last element?

Reply:
We are doing the last element check to save on time. But this is being done for each test (>50) so doesn't make sense to add for this specific test. In that case, should i add it to the top of the function?